llvm/test/Verifier/operand-bundles.ll

```diff
@@ -105,4 +105,17 @@ declare ptr @objc_retainAutoreleasedReturnValue(ptr)
 declare ptr @objc_unsafeClaimAutoreleasedReturnValue(ptr)
 declare void @llvm.assume(i1)
 
+define void @f_type(ptr %ptr) {
+; CHECK: Multiple "callee_type" operand bundles
+; CHECK-NEXT: call void @g() [ "callee_type"(metadata !"_ZTSFvE.generalized"), "callee_type"(metadata !"_ZTSFvE.generalized") ]
+; CHECK-NOT: call void @g() [ "callee_type"(metadata !"_ZTSFvE.generalized") ]
```

Comment:
-NOT checks are too fragile, use comprehensive -NEXT checks

Reply:
Typically, I'd agree. In this case, however, the -NOT check is making sure that no verifier errors are created for the correct instance of the call instruction. Only way I could think of addressing this comment is to remove the NOT check. Is that acceptable to you?

Reply:
At a minimum remove the not check. Do you need CHECK-EMPTY or use FIleCheck -implicit-check-not=error or something similarly broad

Reply:
I can remove the NOT check. I tried CHECK-EMPTY but the OPT tool returns a `error: input module is broken!` error. I am adding a `; CHECK-NEXT: error: input module is broken!` as the last line of the file.